include/idf/CompositeFlightController.hh

```diff
@@ -76,6 +76,28 @@ class CompositeFlightController : public CompositeController<FlightController> {
      */
     double getCommandedZ() const;
 
+    /**
+     * gets the conglomerate trigger value, which is the logical
+     * OR of all constituent controllers' trigger values
+     *
+     * @return the combined trigger of all added robotics controllers
+     */
+    bool getCommandedTrigger() const;
+
+    /**
+     * returns the commanded Comm
+     *
+     * @return the Comm command
+     */
+    bool getCommandedComm() const;
+    
+    /**
+     * returns the commanded shutdown
+     *
+     * @return the shutdown command
+     */
+    bool getCommandedShutdown() const;
```

Comment:
Adding Trigger is fine since it is a generic button definition. But Comm and Shutdown are vehicle/sim specific. You should inherit the flight controller classes into your own controller that has these buttons defined, and keep that as part of your sim.